tests/vmi_lifecycle_test.go

```diff
@@ -1823,6 +1823,75 @@ var _ = Describe("[rfe_id:273][crit:high][arm64][vendor:cnv-qe@redhat.com][level
 			Expect(event).To(BeNil(), "virt-handler tried to sync on a VirtualMachineInstance in final state")
 		})
 	})
+
+	Context("replicaset with topology spread constraints", func() {
+		It("Replicas should be spread across nodes", func() {
```

Comment:
I am not sure we need to test this behavior. I think it is enough to ensure we reflect it on the underlying pod. WDYT?
@vasiliy-ul

Reply:
Hm.. Do you think that this test is just outside the KubeVirt's scope? Probably yes, it verifies the 'spread' functionality which more belongs to K8s in general. I think it won't hurt to have it. Though, I do not have a strong opinion here and do not insist on keeping it.

Or do you have other thoughts in mind?

Reply:
Yes, I think it is a little bit out of scope but I like to test these kinds of stuff as assumptions or software can break. I think we could merge this test with the first one.

Reply:
I'll leave it in then? :-)

Reply:
Fine by me